src/modules/auth/auth.service.ts

```diff
@@ -35,7 +35,7 @@ export class AuthService {
     const accessToken = await this.generateToken('ACCESS', payload);
     const refreshToken = await this.generateToken('REFRESH', payload);
 
-    return { accessToken, refreshToken };
+    return { accessToken, refreshToken, existingUser };
```

Comment:
can you explain why you are returning existingUser here and not createdUser? isn't the value of the existingUser always null because you are throwing an exception if it is truthy above.

Reply:
removed that